src/main/java/ch/admin/bag/covidcertificate/sdk/core/extensions/VaccinationEntryExtensions.kt

```diff
@@ -68,3 +70,7 @@ fun VaccinationEntry.getIssuer(): String {
 fun VaccinationEntry.getCertificateIdentifier(): String {
 	return this.certificateIdentifier
 }
+
+fun TestEntry.isTouristVaccine(): Boolean {
+	return this.type == TestType.SEROLOGICAL.code
```

Comment:
this looks like a copy&paste error to me

Reply:
ist nicht mehr nötig, hab ich vergessen zu entfernen